test/ibmq/test_ibmq_job.py

```diff
@@ -500,8 +445,7 @@ def test_retrieve_from_retired_backend(self):
             self.assertTrue(isinstance(new_job.backend(), IBMQRetiredBackend))
             self.assertNotEqual(new_job.backend().name(), 'unknown')
 
-            new_job2 = self.provider.backend.jobs(
-                db_filter={'id': self.sim_job.job_id()}, start_datetime=self.last_month)[0]
+            new_job2 = self.provider.backend.jobs(start_datetime=self.last_month)[0]
```

Comment:
Now that we don't pass job_id in db_filter, is the first job (index 0) guaranteed to be new_job2 ? Instead could we check if new_job2 is in list of jobs retrieved from last month?

Reply:
not sure what you mean here - we are setting `new_job2` equal to the first job from the last month - of course it'll be in the list of jobs from the last month right?

Reply:
It should be in the list but is it guaranteed to be the first in the list? you are looking at index 0 - `self.provider.backend.jobs(start_datetime=self.last_month)[0]`